### DISCUSSION TITLE
NuGet links need updated

### DISCUSSION THREAD

**Discussion by webprofusion-chrisc** [bcgit/bc-csharp#538] [General]
Hi,

If you look at the NuGet package here: https://www.nuget.org/packages/BouncyCastle.Cryptography/

The links for Project Website [and more usefully, the Release Notes] go to a 404 on bouncycastle.org - this probably needs an update to the nuget package or just a redirection setup on the website.

**Comment by peterdettman** [comment]
Thanks for the report, we've set up redirects now.
